cookbook/src/components/Footer/FooterComponent.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Container, LinkStyled } from '../common/StylesComponent';
+import { FooterLogo, FooterStyle, LinksContainer } from './style/FooterComponentStyle';
+
+const Footer = () => {
+  return (
+    <>
+      <Container color="var(--light-black)" justifyContent="center">
+        <FooterStyle color="var(--light-black)">
+          <Container>
+            <FooterLogo />
+          </Container>
+
+          <LinksContainer>
+            <LinkStyled>Cookbooks</LinkStyled>
+            <LinkStyled>Recepies</LinkStyled>
+            <LinkStyled>About Us</LinkStyled>
+            <LinkStyled>Cookbooks@gmail.com</LinkStyled>
```

Comment:
let's use the prefix 'mailto' for mail links, example:  `href= "mailto:Cookbooks@gmail.com"`

Reply:
I added functionality of 'href' to links in new update. Here custom href attribute is is unavailible. May I add it in next pr?

Reply:
ok